contracts/action-handlers/AddRewardRuleHandler.sol

```diff
@@ -41,9 +41,26 @@ contract AddRewardRuleHandler is Ownable, Versionable {
     return true;
   }
 
+  /**
+   * @dev onTokenTransfer(ERC677) - this is the ERC677 token transfer callback.
+   *
+   * This adds the reward rule to the reward manager.
+   *
+   * See AddRewardRuleHandler in README for more information.
+   *
+   * @param from the token sender (should be the action dispatcher)
+   * @param data encoded as: (
+   *  address prepaidCard,
+   *  uint256 ???,
```

Comment:
@tintinthong here in [this line](https://github.com/cardstack/card-pay-protocol/blob/main/test/RewardManager-test.js#L486) it says it doesn't matter what this is - how come we can't just remove it?

Reply:
Owh this was in the test right. Yea just remove all instance of "it doesn't matter what it is". I think it was me simulating a transferAndCall but the handler doesn't care about the amount.